### ISSUE TITLE
Customization process doesn't reboot system

### ISSUE THREAD

**Issue by alexandrovas** [vmware/open-vm-tools#458]
OS: Ubuntu Server 18.04.5 (all packages upgraded)
open-vm-tools version: 11.0.5 (from ubuntu repository)
vCloud Director version: 9.7.0.16081827

After changing hostname and network configuration host is not restarting.
Error in log file /var/log/vmware-imc/toolsDeployPkg.log:
```
[   error] Customization command failed with stderr: 'Failed to create bus connection: No such file or directory

```

After changing file /lib/systemd/system/open-vm-tools.service (adding line **After=dbus.service**) the reboot is working. 

This article https://kb.vmware.com/s/article/56409 described the workaround and indicated that this is fixed in newer versions. But i see that this is not fixed in my version 11.0.5.

**Comment by PengpengSun** [comment]
Hi @alexandrovas 

This fix on dbus.service dependence is in vSphere 6.7 U3 and vSphere 6.5 P04.
The fix on /tmp is in open-vm-tools version 10.3.10 and above.
Yes, the KB describes the workaround for both above issues.

You are using vCD 9.7.0 which code base is vSphere6.5 P03, so it does NOT have the dbus.service dependence fix.

> After changing file /lib/systemd/system/open-vm-tools.service (adding line After=dbus.service) the reboot is working.

Please apply the workaround for now as you did.

**Comment by stanguturi** [comment]
Thanks @PengpengSun The instructions mentioned in the previous update should fix the issue. Closing the issue.
